services/ui-src/src/components/reports/ReportPageFooter.tsx

```diff
@@ -10,18 +10,14 @@ import { FormJson } from "types";
 import nextIcon from "assets/icons/icon_next_white.png";
 import previousIcon from "assets/icons/icon_previous_blue.png";
 
-export const ReportPageFooter = ({
-  submitting,
-  form,
-  hidePrevious,
-  ...props
-}: Props) => {
+export const ReportPageFooter = ({ submitting, form, ...props }: Props) => {
   const navigate = useNavigate();
   const { report } = useContext(ReportContext);
   const { previousRoute, nextRoute } = useFindRoute(
     report?.formTemplate.flatRoutes,
     report?.formTemplate.basePath
   );
+  const hidePrevious = previousRoute === "/mcpar" || previousRoute === "/mlr";
```

Comment:
Turns out all of MLR was getting no Previous button. I've re-added it and made it so that the first page of the form will not have a previous button but all others will. That is based on the mockups and [feedback from Megan here](https://coforma.slack.com/archives/C031DGE8W7Q/p1690573250353099?thread_ts=1690573137.945089&cid=C031DGE8W7Q)